### DISCUSSION TITLE
How to access parent value in nested creation of update()?

### DISCUSSION THREAD

**Discussion by Yiak** [prisma/prisma#17465] [Q&A]
The `topic` model has a `comments_count` field to record how many comments under this topic. The comment model also has a `comment_number` in itself (served as an index).

When a new comment was created with `prisma.topic.update()`, I would like to assign the comment number to the newly created comment. How could I do this?

```ts
const topic = await prisma.topic.update({
    where: {
        id: topic_id
    },
    data: {
        comments: {
            create: {
                content,
                comment_number // <= I want to assign the comments_count from topic model to this field
	    }
	},			
	comments_count: {
		increment: 1
	}
    }
});
```

**Comment by nurul3101** [comment]
Hello @Yiak 👋 

I don't think this would be possible in a single query, you would need to use an [Interactive transaction](https://www.prisma.io/docs/concepts/components/prisma-client/transactions#interactive-transactions) to get the comments_count value first and then use it in the next update query.

We do have a preview feature for [fieldReference](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#compare-columns-in-the-same-table) to compare values from the same table but that wouldn't help in your use case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
